src/Numerical_aux/typesdef.f90

```diff
@@ -931,7 +931,22 @@ REAL                         , POINTER :: BF_GP_Hex(:,:)                   !<Val
      ! case(6) bimaterial with LSW
      REAL                                   :: v_Star                           !< reference velocity of prakash-cliff regularization
      REAL                                   :: L                                !< reference length of prakash-cliff regularization
-     REAL, POINTER                          :: Strength(:,:) => NULL()                   !< save strength since it is used for bimaterial
+     REAL, POINTER                          :: Strength(:,:) => NULL()          !< save strength since it is used for bimaterial
+     INTEGER                                :: thermalPress                     !< thermal pressurization switch
+     REAL                                   :: alpha_th                         !< thermal diffusion parameter for TP
+     REAL                                   :: alpha_hy                         !< hydraulic diffusion parameter for TP
+     REAL                                   :: rho_c                            !< heat capacity for TP
+     REAL                                   :: TP_lambda                        !< pore pressure increase per unit increase
+     INTEGER                                :: TP_nz                            !< number of grid points to solve advection for TP
+     REAL                                   :: TP_dlDwn                         !< grid space distance for TP_grid
```

Comment:
Why not using more explicit variable names?
DISC%DynRup%TP_hwid vs TP_grid_half_width?
DISC%DynRup%TP_dlDwn vs TP_grid_dz? or TP_dz? given the fact that there is nz?
DISC%DynRup%TP_nz vs TP_grid_nz?
DISC%DynRup%TP_Dwnmax vs DISC%DynRup%TP_max_wavenumber
update_RSF vs update_State_Variable_RSF

Reply:
the name conventions were introduced by Stephanie. I didn't replace it. 
but those ones are short and still explainable

Reply:
I agree with Thomas. We are not in the 70s anymore where we need cryptic abbreviations to save a few bytes.

Reply:
Hi,
So after working on TP, I realized that the notations are the same as in the matlab script on the TP SCEC benchmark.  Therefore, it might make sense to keep them.

Reply:
Just because other people use bad naming doesn't imply that we should also use bad naming.
Call the variable TP_max_wavenumber and (maybe) add a comment that it corresponds to TP_Dwnmax.